src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -157,15 +160,16 @@ const ShareInviteDialog = (props: ShareInviteDialogProps): JSX.Element => {
     let isThereAnyNewUser = newUsersExists;
 
     if (usersList.length === 0 && isValidEmail(email)) {
-      const publicKey = await getUserPublicKey(email);
-      if (!publicKey && !preCreateUsers) {
+      const { publicKey, publicKyberKey } = await getUserPublicKey(email);
+      if (!publicKey && !preCreateUsers && !publicKyberKey) {
         isThereAnyNewUser = true;
       }
       usersList.push({
         email,
         userRole,
-        isNewUser: !publicKey,
+        isNewUser: !publicKey || !publicKyberKey,
```

Comment:
The user will have both keys always?

Reply:
That's the part I'm still thinking about - migration, when we only will have 1 key out of 2